string new_msg
---
bool response